crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/ContributionService.java

```diff
@@ -112,17 +112,19 @@ public ContributionResponseDTO checkContribsCondition(ContributionRequestDTO req
                 contributionResponse.setReassessmentCoteId(contributionResponseDTO.getReassessmentCoteId());
                 contributionResponse.setDoContribs(contributionResponseDTO.getDoContribs());
                 if (CorrespondenceType.getFrom(processedCases.getCotyCorrespondenceType()) != null) {
-                    contributionResponse.setCorrespondenceTypeDesc(CorrespondenceType.getFrom(processedCases.getCotyCorrespondenceType()).getDescription());
+                    contributionResponse.setCorrespondenceTypeDesc(
+                            Objects.requireNonNull(CorrespondenceType.getFrom(processedCases.getCotyCorrespondenceType())).getDescription());
```

Comment:
CorrepondenceType.getFrom throws IllegalArgumentException if not found. What is the significance of Objects.requireNonNull?